src/order.rs

```diff
@@ -0,0 +1,44 @@
+use async_graphql::{Enum, SimpleObject};
+use bson::Uuid;
+use serde::{Deserialize, Serialize};
+
+use crate::{
+    foreign_types::{User, VendorAddress},
+    http_event_service::OrderEventData,
+    invoice::Invoice,
+};
+
+/// Foreign type of an order.
+#[derive(Debug, Serialize, Deserialize, SimpleObject, Clone)]
+#[graphql(unresolvable = "id invoice { id }")]
```

Comment:
you sure that this is what we want/need here?

Reply:
As i understood it this is the way to add a field to an entity defined in another service, but correct me if this is not what we want here

Reply:
Otherwise i can take this out if we not necessarily need it and it creates problems with the schemas